src/components/TopicLoader/TopicLoader.js

```diff
@@ -294,7 +301,7 @@ class TopicLoader extends Component {
         flatLayers[i].getApiKey &&
         !flatLayers[i].getApiKey()
       ) {
-        flatLayers[i].setApiKey(apiKey);
+        flatLayers[i].setApiKey(apiKey, apiKeyName);
```

Comment:
As far as I know, this is currently only relevant for react-transit TrajservLayer (see https://github.com/geops/react-transit/blob/master/src/layers/TrajservLayer.js#L484). I will ask in the ticket, if react-transit changes are necessary. Even if not, I would pass the apiKeyName here for future layers and consistency.

Reply:
Done.